### DISCUSSION TITLE
What will happen to my github copilot access

### DISCUSSION THREAD

**Discussion by Phoenix-rat** [community/community#29437] [Copilot Conversations]
What will happen to my github copilot access?
Will beta testers still be able to use copilot after Copilot is public and pricing is introduced?

**Comment by davecheney** [comment]
> Will beta testers still be able to use copilot after Copilot is public and pricing is introduced?

By August 22, you must choose either a monthly or yearly plan to continue to use Copilot.

**Comment by softwaredevelop** [comment]
or

> A free subscription for GitHub Copilot is available to verified students, and maintainers of popular open-source repositories on GitHub.

https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot#github-copilot-pricing
